cmd/daemon_logger/main.go

```diff
@@ -0,0 +1,74 @@
+package main
+
+import (
+	"flag"
+	"strings"
+	"time"
+
+	"github.com/DataDog/datadog-go/statsd"
+	"github.com/Sirupsen/logrus"
+)
+
+var (
+	hostport  = flag.String("hostport", "", "Hostname and port of destination.")
```

Comment:
What part of the config file do we want to use? Is it the [`udp_address`?](https://github.com/stripe/veneur/blob/master/example.yaml#L27)